notary-integration-test/src/main/kotlin/integration/helper/AccountHelper.kt

```diff
@@ -52,13 +52,13 @@ class AccountHelper {
     val tokenStorageAccount = notaryAccount
 
     /** Account that sets tokens */
-    val tokenSetterAccount by lazy { createTesterAccount("eth_tokens", "token_service") }
+    val tokenSetterAccount by lazy { createTesterAccount("eth_tokens", "eth_token_list_storage") }
 
     // TODO - D3-348 - dolgopolov.work change to suitable role name
```

Comment:
```suggestion
    // TODO - D3-348 - dolgopolov.work change to suitable role name
```
You can remove it now